lib/usd/hdMaya/adapters/proxyAdapter.cpp

```diff
@@ -61,18 +61,11 @@ void HdMayaProxyAdapter::Populate() {
     TF_DEBUG(HDMAYA_AL_POPULATE)
         .Msg("HdMayaProxyDelegate::Populating %s\n", _proxy->name().asChar());
 
-    auto stage = _proxy->getUsdStage();
```

Comment:
It's true that the base proxy shape currently always has a valid stage. This doesn't have to be true a) in future, b) for plugin proxy shapes (example AL proxy shape). Please bring back this validation.